src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+
+import { Filters } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  filter: Filters;
+  setFilter: (str: Filters) => void;
+  deleteCompletedTodo: () => void;
+}
+
+export const TodoFooter = ({
+  todos,
+  filter,
+  setFilter,
+  deleteCompletedTodo,
+}: Props) => {
+  const activeTodosLength = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodosLength} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        {Object.values(Filters).map((item, i) => {
+          return (
+            <a
```

Comment:
Remove index, use item as `key` value
```suggestion
        {Object.values(Filters).map((item) => (
            <a
```